test/service.thrift

```diff
@@ -0,0 +1,7 @@
+
+service Foo {
+    byte foo(0: byte number) throws (
```

Comment:
Ah, I’ll forbid field identifier 0 on structs, except for the generated success field.